TROUBLESHOOTING.md

```diff
@@ -0,0 +1,70 @@
+Troubleshooting the plugin
+==============
+
+### Verifying the setup
+
+**0.** ensure the latest compatible version of IntelliJ IDEA Community Edition is installed (**2020.2.** or later).
+
+**1.** update plugin to the latest version (you can check the latest version on [Plugin's official page](https://plugins.jetbrains.com/plugin/13634-a-courses/versions)), but at least **1.4**.
+
+**2.** create a new project: [**File -> New -> Project...** and then **Empty Project**](https://plus.cs.aalto.fi/o1/2020/w01/ch02/#launch-intellij).
+
+**3.** Turn project into an A+ Project: [**A+ -> Turn Project Into A+ Project**](https://plus.cs.aalto.fi/o1/2020/w01/ch02/#the-a-courses-plugin) (and choose the rest of the settings asked for in the pop-up dialog).
```

Comment:
Is this meant to be done always? I think it would be good to first try to fix the issues in the existing project

Reply:
this is our analogy to the "please restart your computer", so yes, I'd like this to be here. there is a myriad of possibilities why their initial setup went sour, missed a step, some network issues or somesuch (that you'll never get to reproduce). whereas, running the script through verifies, that the whole thing can be run in that particular environment. once done, then it would make sense to a) use this new project as a base for the course, b) if curious/have time - research what was the cause of trouble